tokens/tokens-eth.json

```diff
@@ -2304,6 +2304,38 @@
   "youtube"   : ""
 }
 },{
+"symbol"      : "BUC",
+"address"     : "0xca3c18a65b802ec267f8f4802545e7f53d24c75e",
+"decimals"    : "18",
+"name"        : "BeeUnity Chain",
+"ens_address" : "",
+"website"     : "https://beeunity.org",
+"logo": {
+  "src"       : "https://",
```

Comment:
Can you remove the logo source or add the proper link if there is one.

Reply:
Sure! Thanks